mm2src/coins/utxo/slp.rs

```diff
@@ -1661,7 +1663,7 @@ impl MmCoin for SlpToken {
                 WithdrawError::from_generate_tx_error(gen_tx_error, coin.platform_ticker().into(), platform_decimals)
             })?;
 
-            let prev_script = ScriptBuilder::build_p2pkh(&my_address.hash);
+            let prev_script = output_script(my_address);
```

Comment:
Same here, it seems to break signing if Segwit is used (it's not supported for BCH/SLP, though).

Reply:
I guess segwit address will never be created based on coin data?

Reply:
Yes, it won't be created unless BCH supports Segwit one day 🙂